qml/pages/ProgramDelegate.qml

```diff
@@ -26,15 +27,18 @@ ListItem {
     }
 
     Item {
-        height: parent.height - 2 * Theme.paddingSmall
+        id: listContainer
+        //height: parent.height - 2 * Theme.paddingSmall
+        height: title.height + episode.height + time.height + 2*Theme.paddingSmall
```

Comment:
It would make these more simple if you add 2 * Theme.paddingSmall to listItem contentHeight and then matching anchor margins to this listContainer and then remove the top margins from elsewhere.

Reply:
Thanks, I'll have a look at simplifying the layout calculations again.